README.md

```diff
@@ -284,7 +284,7 @@ these are skipped by default during CI runs. If you also do not want to run thes
 they can simply be skipped like this:
 
 ```bash
-$ php vendor/bin/phpunit --exclude-group internet
+php vendor/bin/phpunit --exclude-group internet
```

Comment:
Looks like a good opportunity to also get rid of the `php` binary here for consistency reasons (not needed and also not used in most components) :+1: